Gadget.Server/Domain/Entities/Service.cs

```diff
@@ -25,6 +25,9 @@ private Service()
         public string LogOnAs { get; }
         public string Description { get; }
         public Agent Agent { get; }
+
+        public Guid AgentId { get; set; }
```

Comment:
Wolalbym shadow property niz trzymac to w modelu bizensowym, nie mieszajmy db z domena